packages-content-model/roosterjs-content-model-plugins/lib/edit/tabUtils/handleTabOnParagraph.ts

```diff
@@ -0,0 +1,75 @@
+import { createSelectionMarker, createText } from 'roosterjs-content-model-dom';
+import { setModelIndentation } from 'roosterjs-content-model-api';
+import type {
+    ContentModelDocument,
+    ContentModelParagraph,
+    RangeSelection,
+} from 'roosterjs-content-model-types';
+
+const tabSpaces = '    ';
+const space = ' ';
+
+/**
+ * @internal
+ */
+export function handleTabOnParagraph(
```

Comment:
Please copy the behavior specification (listed in PR description) here so that future we can still know the expected behavior